### ISSUE TITLE
No Thumbnails

### ISSUE THREAD

**Issue by imurreflection** [dobbelina/repository.dobbelina#244]
Many sites are not showing thumbnails. I m checking all sites now but so far, following sites have no Thumbnails:

Porn00
ViralVideosPorno
Mr Sexe
Ipornovideos

Kindly check

**Comment by jdoedev123** [comment]
I have Thumbnails for all of the sites mentioned above.
Not sure what your problem is. I don't know if it is related but I just fixed a problem of an expired Certificate ( AddTrust_External_Root ) which has been breaking many things on the Internet since it expired May 30, 2020. For me it broke HQPorner, but I did not check all the other sites. Some of my computers updated themselves to the fix the problem while others needed to be manually fixed.
